### PR TITLE
PR Review Changes for CR #9 Banner Highlights

### PR DESCRIPTION
**Pull request by uptownhr**
- [x] awaiting SVG for arrow
- [x] remove breakpoints PR feedback

### CODE REVIEW

**Comment by uptownhr** [comment]
drew on the previous PR that I accidentally closed, you menttioned 
image

You mentioned that breakpoints are not needed but removing the breakpoints cause the mobile view to look quite off. Did you mean that you'd like me to take another approach so it can be responsive without breakpoints?

Without breakpoints. Padding and font-size seems off.
image

File: src/components/BannerHighlight/BannerHighlight.vue [hunk 1]
```diff
@@ -12,8 +12,8 @@
             <!-- prompt -->
 
             <div class="prompt-container">
-                <p class="prompt" v-html="prompt" />
-                <span class="arrow">Arrow</span>
+                <span class="prompt" v-html="prompt" />
+                <arrow class="arrow" />
```
Comment (drewbaker) [inline src/components/BannerHighlight/BannerHighlight.vue:16]: Can we call this <svg-arrow> and SvgArrow, thanks!

FYI This is an example of something I'd love to just be able to change myself easily.

File: src/components/BannerHighlight/BannerHighlight.vue [hunk 2]
```diff
@@ -12,8 +12,8 @@
             <!-- prompt -->
```
Comment (drewbaker) [inline src/components/BannerHighlight/BannerHighlight.vue:12]: Don't need this

File: src/components/BannerHighlight/BannerHighlight.vue [hunk 3]
```diff
@@ -77,45 +79,35 @@ export default {
...
+            transition: transform 200ms ease;
         }
 
-        &:hover .arrow {
-            left: 100%;
+        @media #{$has-hover} {
+            &:hover .arrow {
+                left: 100%;
+                transform: translateX(30%)
+            }
```
Comment (drewbaker) [inline src/components/BannerHighlight/BannerHighlight.vue:104]: The hover state would be the entire element. The arrow moves to indicate that a link can be clicked on,  as the entire element is the link, it should move when hover on it.

Comment (uptownhr) [reply src/components/BannerHighlight/BannerHighlight.vue]: hover added to entire element

File: src/components/BannerHighlight/BannerHighlight.vue (hunk 2 above)
Comment (uptownhr) [reply src/components/BannerHighlight/BannerHighlight.vue]: removed

File: src/components/BannerHighlight/BannerHighlight.vue (hunk 1 above)
Comment (uptownhr) [reply src/components/BannerHighlight/BannerHighlight.vue]: updated

**Comment by drewbaker** [comment]
SVG should go in `@/assets/svg` as documented here: https://github.com/funkhaus/factory#svgs

This is nit-picky I know, but hover state should go above breakpoints (in theory you might want to over ride hovers at certain breakpoints - it's just a good habit to enforce).

You're also importing `formatDate` for no reason it seems?

**Comment by uptownhr** [comment]
@drewbaker moved the hover definition above the breakpoints and also removed the unused format date helper. However, I think it's worth while discussing where the SVG should live. I'd vote to keep this in the component. When the component gets used in the project site, it'll be much easier that it's together. Or else you'll have to also copy over the assets directory.

**Pull request merged**